docs/source/udf.rst

```diff
@@ -81,6 +81,8 @@ Auto UDF
 
 The :class:`~libertem.udf.AutoUDF` class and :meth:`~libertem.api.Context.map` method allow to run simple functions that accept a frame as the only parameter with an auto-generated :code:`kind="nav"` result buffer over a dataset ad-hoc without defining an UDF class. For more advanced processing, such as custom merge functions, post-processing or performance optimization through tiled processing, defining an UDF class is required.
 
+As an alternative to Auto UDF, you can use the :meth:`~libertem.io.dataset.base.DataSet.get_dask_array` method of a :class:`~libertem.io.dataset.base.DataSet` and use a `dask.array <https://docs.dask.org/en/latest/array.html>`_ to perform your calculations. See :ref:`Integration with Dask arrays<daskarray>` for more details.
```

Comment:
Needs to be updated to point to function instead of method, looks good otherwise!

Reply:
Thx, nice catch, fixed!